scripts/prep-ww-validation.py

```diff
@@ -121,64 +140,63 @@ def descends_from_target(taxid, cache={}):
                 (query_qual_fwd, query_qual_rev),
             ):
                 if taxid == SARS_COV_2_TAXID:
-                    sars_reads.append(
-                        (taxid, seq, qual, date, fine_location, read_id, sample_id)
+                    sars_reads[sample_delivery].append(
```

Comment:
nit: this would be slightly clearer as:

```
record = taxid, seq, qual, date, fine_location, read_id, sample_id, is_duplicate
if taxid == SARS_COV_2_TAXID:
  sars_reads[sample_delivery].append(record)
else:
  to_validate[sample_delivery].append(record)
```